entrypoint.sh

```diff
@@ -15,17 +15,50 @@ EXTERNAL_ID="${EXTERNAL_ID:-${GROUP_ID}}"
 DB_HOST="${DB_HOST:-localhost}"
 DB_PORT="${DB_PORT}"
 DB_TYPE="${DB_TYPE:-postgres}"
-DB_NAME="${DB_NAME:-DB_NAME}"
-DB_USER="${DB_USER:-DB_USER}"
-DB_PASS="${DB_PASS:-DB_PASS}"
-USERNAME="${USERNAME:-''}"
-PASSWORD="${PASSWORD:-''}"
+DB_NAME="${DB_NAME}"
+DB_USER="${DB_USER}"
+DB_PASS="${DB_PASS}"
+USERNAME="${USERNAME}"
+PASSWORD="${PASSWORD}"
 SYNC_URL="${SYNC_URL}"
 REGISTRATION_URL="${REGISTRATION_URL}"
 REPLICATE_TO="${REPLICATE_TO}"
 REPLICATE_TABLES="${REPLICATE_TABLES}"
 REPLICATE_COLS=""
 
+# exit on no registration_url
+if [[ "$REGISTRATION_URL" != "" ]]; then
+  echo "You must set REGISTRATION_URL" && exit 1;
+fi
+
+# exit on no replicate_tables
+if [[ "$REPLICATE_TABLES" != "" ]]; then
+  echo "You must set REPLICATE_TABLES" && exit 1;
+fi
+
+# exit on no sync_url
+if [[ "$SYNC_URL" != "" ]]; then
+  echo "You must set SYNC_URL" && exit 1;
+fi
+
+# exit on no replicate_to
+if [[ "$REPLICATE_TO" != "" ]]; then
+  echo "You must set REPLICATE_TO" && exit 1;
+fi
+
+# exit if db creds have not been set
+if [[ "$DB_NAME" != "" ]] || [[ "$DB_USER" != "" ]] || [[ "$DB_PASS" != "" ]] || [[ "$DB_PORT" != "" ]]; then
+  echo "You must set DB_NAME, DB_USER and DB_PASS" && exit 1;
+fi
```

Comment:
I'd be inclined to separate these out like everything else just so that they know what they are missing.

What do you think?

What we could do on all these error cases is call a `usage` function that echos an outline of which environment variables can be set. (A bit like the README does.)